src/port1.0/porttrace.tcl

```diff
@@ -207,6 +207,9 @@ namespace eval porttrace {
             }
         }
 
+		# Allow access to SDK if it's not inside the Developer folder.
```

Comment:
Indent looks off. Please use spaces instead of tabs to be consistent with the rest of the file.

Reply:
Sorry I prefer tabs and forget to convert to spaces half the time before submitting 😅

Didn't set configure.sdk in the portfile but rather in macports.conf (and sometimes also via the command-line). This is sometimes necessary when compiling binaries meant to be compatible with older systems because else you might get weak-linked symbols that will not exist on older versions of macOS. And the reason I didn't put the SDK under the developer directory is that I'm under the impression that folder is nuked whenever Xcode is updated (and technically speaking, Apple does not even ship the 10.12 SDK anymore, imagine ones older than that)